scripts/src/main/resources/scripts/devon.properties

```diff
@@ -15,6 +15,8 @@ export JAVA_HOME=${DEVON_IDE_HOME}/software/java
 #export M2_REPO=~/.m2/repository
 export M2_REPO=${DEVON_IDE_HOME}/conf/.m2/repository
 export MAVEN_HOME=${DEVON_IDE_HOME}/software/maven
+# CobiGen CLI needs this
+export M2_HOME=${DEVON_IDE_HOME}/software/maven
 export MAVEN_OPTS=-Xmx512m -Duser.home=${DEVON_IDE_HOME}/conf
```

Comment:
I have found a way to programmatically set `MAVEN_HOME` instead of `M2_HOME` inside the CobiGen CLI. I have tested it and it should not be a problem now.